ui/ui.go

```diff
@@ -73,14 +73,11 @@ func renderForDev(w io.Writer, root string, c *content, cfg *config.Config, r *h
 	}
 
 	var buf bytes.Buffer
-	fmt.Fprintf(
-		&buf,
-		"<script src=\"/js/JSXTransformer-%s.js\"></script>\n",
```

Comment:
Right. JSXTransformer was previously only added in Dev mode because Dev mode was the only time we needed runtime transformation of JSX. In Prod mode, we used the command line `jsx` to bundle raw javascript in bindata.go (this is located in the `Makefile`).